Maple2.Server.Game/Util/ConditionUtil.cs

```diff
@@ -13,6 +13,11 @@ public static bool Check(this ConditionMetadata condition, GameSession session,
 
     private static bool CheckCode(this ConditionMetadata.Parameters code, GameSession session, ConditionType conditionType, string stringValue = "", long longValue = 0) {
         switch (conditionType) {
+            case ConditionType.emotiontime:
+                if (code.Strings != null && code.Strings.Contains(stringValue)) {
+                    return true;
+                }
+                break;
```

Comment:
Just add a new case, no need to repeat the check
```suggestion
            case ConditionType.emotiontime:
```